zulipterminal/helper.py

```diff
@@ -271,6 +274,44 @@ def set_count(id_list: List[int], controller: Any, new_count: int) -> None:
     controller.update_screen()
 
 
+def analyse_edit_history(
+    msg_id: int,
+    index: Index,
+    content_changed: bool,
+    stream_changed: bool,
+    current_topic: Any = None,
+    old_topic: Any = None,
+) -> None:
+    resolve_change = False
+    resolved_prefix = RESOLVED_TOPIC_PREFIX + " "
+    if content_changed or stream_changed:
+        index["edited_messages"].add(msg_id)
+    elif old_topic:
```

Comment:
This also depends upon current_topic being non-None, which as it stands is not guaranteed. In other similar cases we have an assert at the top of the function to ensure that 'coordinated' values are used correctly. The alternative is to define the parameter as a data structure.